src/Files.Uwp/Views/Pages/Properties.xaml

```diff
@@ -94,6 +99,7 @@
                     AccessKey="G"
                     Content="{helpers:ResourceString Name=General}"
                     CornerRadius="0"
+                    d:IsSelected="True"
```

Comment:
Is this necessary? Trying to keep the xaml files as slim as possible for better readability.

Reply:
Not really, it just gives a better idea about what tab is selected by default in the designer.

Reply:
I recommend removing it but it's up to you

Reply:
Sure thing, I was just testing to see how it looks in the designer